test/junit/scala/runtime/ScalaRunTimeTest.scala

```diff
@@ -27,6 +27,8 @@ class ScalaRunTimeTest {
     assertEquals("""Array(Array("", 1, Array(5)), Array(1))""",
         stringOf(Array(Array("", 1, Array(5)), Array(1))))
 
+    assertEquals("""ArraySeq()""", stringOf(collection.mutable.ArraySeq.empty[AnyRef]))
+
```

Comment:
why is this in `ScalaRunTimeTest`?

Reply:
I added it to test something, but don't recall what.  I'll remove.

Reply:
the `MapView` one I don't think should be here either. I think there's already a `MapViewTest` maybe?

Reply:
Good to know.  I've worked on this in fits and starts, so might have not existed when I began.